src/provider_misc.jl

```diff
@@ -16,8 +16,8 @@ function process(r::JSONRPC.Request{Val{Symbol("initialize")},Dict{String,Any}},
         for (root, dirs, files) in walkdir(server.rootPath)
             for file in files
                 if splitext(file)[2]==".jl"
-                    filepath = joinpath(root, file)
-                    uri = string("file:///", replace(replace(filepath, '\\', '/'), ":", "%3A"))
+                    filepath = joinpath(root, file)
+                    uri = string("file:///", is_windows() ? replace(replace(filepath, '\\', '/'), ":", "%3A") : filepath)
```

Comment:
still one too many slashes, on linux the filepaths lead with a forward slash so we end up with `file:////blah/blah`